api/v3/CxnApp.php

```diff
@@ -25,8 +25,8 @@
  +--------------------------------------------------------------------+
  */
 
-use \Civi\Cxn\Rpc\Message\AppMetasMessage;
-use \Civi\Cxn\Rpc\Message\GarbledMessage;
+use Civi\Cxn\Rpc\Message\AppMetasMessage;
```

Comment:
@totten you may have a feeling on this but i think this is probably ok

Reply:
Yeah, I think it's OK.